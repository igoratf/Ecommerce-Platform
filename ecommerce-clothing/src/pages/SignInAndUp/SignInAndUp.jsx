import React from 'react';
import './SignInAndUp.scss';
import SignIn from '../../components/SignIn/SignIn';

const SignInAndUp = () => (
    <div className="sign-in-and-up">
        <SignIn />
    </div>
)

export default SignInAndUp;