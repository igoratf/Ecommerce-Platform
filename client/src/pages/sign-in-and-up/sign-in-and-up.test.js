import { shallow } from 'enzyme';
import React from 'react';
import SignInAndUp from './sign-in-and-up.component';

it('expect to render sign in and sign up page', () => {
  expect(shallow(<SignInAndUp />)).toMatchSnapshot();
})
