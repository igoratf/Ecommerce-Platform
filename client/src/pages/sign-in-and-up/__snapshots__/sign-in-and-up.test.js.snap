// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render sign in and sign up page 1`] = `ShallowWrapper {}`;
